identity_iota/Cargo.toml

```diff
@@ -8,23 +8,34 @@ keywords = ["iota", "tangle", "identity", "did", "ssi"]
 license = "Apache-2.0"
 readme = "./README.md"
 repository = "https://github.com/iotaledger/identity.rs"
-rust-version = "1.60"
+rust-version = "1.62"
 description = "Framework for Self-Sovereign Identity with IOTA DID."
 
 [dependencies]
 identity_agent = { version = "=0.6.0", path = "../identity_agent", default-features = false, optional = true }
 identity_core = { version = "=0.6.0", path = "../identity_core", default-features = false }
 identity_credential = { version = "=0.6.0", path = "../identity_credential", features = ["validator"], default-features = false }
 identity_did = { version = "=0.6.0", path = "../identity_did", default-features = false }
+identity_iota_core = { version = "=0.6.0", path = "../identity_iota_core", default-features = false }
 
 [dev-dependencies]
+iota-client = { version = "2.0.0-beta.3", default-features = false, features = ["tls", "stronghold"] }
```

Comment:
I don't think this dev-dependency is necessary? I don't even think `tokio` is needed here.

```suggestion
```

Reply:
It's required for the example in the README: https://github.com/iotaledger/identity.rs/blob/chore/stardust-refactor-part-2/identity_iota/README.md#example-creating-an-identity
It's checked for compilation but not run. Since it's not an exact copy of the `create_did` example, it seems like a good idea to keep the check.